Libraries/src/Amazon.Lambda.Annotations/README.md

```diff
@@ -72,6 +73,83 @@ Lambda Annotations uses C# source generators to generate that boiler plate code
 In addition the source generator also synchronizes the CloudFormation template to declare all of the .NET methods with the `LambdaFunction` attribute as 
 Lambda functions in the CloudFormation template.
 
+## Getting started
+
+To get started with Lambda annotations a Lambda blueprint is available. For Visual Studio users the blueprint can be 
+accessed using the [AWS Toolkit for Visual Studio](https://marketplace.visualstudio.com/items?itemName=AmazonWebServices.AWSToolkitforVisualStudio2022). 
+For non-Visual Studio users the [Amazon.Lambda.Templates](https://www.nuget.org/packages/Amazon.Lambda.Templates) 
+NuGet package is available for creating .NET Lambda projects from the .NET CLI.
+
+
+### Visual Studio 2022
+
+To get started with Visual Studio install [AWS Toolkit for Visual Studio](https://marketplace.visualstudio.com/items?itemName=AmazonWebServices.AWSToolkitforVisualStudio2022) 
+extension. Once installed a project can be created with the Lambda Annotations preconfigured using
+the following steps.
+
+* Select **Create a new project**
+* In the template search box enter **AWS Serverless**
+* Select the **AWS Serverless Application (.NET Core C#)** from the search result and click **Next**
+* Name the project and click **Create**
+* The **Select Blueprint** wizard will be displayed for choosing the initial content of the project.
+* Select the **Annotations Framework** blueprint and click **Finish**
+
+### .NET CLI
+
+.NET Lambda projects can be be created using the .NET CLI's `dotnet new` command. To create a project using the
+Lambda Annotations library run the following steps from a terminal.
+
+* Run the following command to install the AWS Lambda templates into the .NET CLI
```

Comment:
I went back and forth on this. I thought if it was on a separate line it would be easier to copy and paste the command. I don't feel strongly about it thought and switched to inline.